config/intellij-idea-inspections.xml

```diff
@@ -1335,9 +1335,9 @@
                      enabled_by_default="true"/>
     <!-- we do like constants on the right side, it is more readable,
          placing constant on the left does not give benefit -->
-    <inspection_tool class="ConstantOnRHSOfComparison" enabled="false" level="ERROR"
```

Comment:
Fixes `Constant on the wrong side of comparison (Errors) (2247)`

Reply:
> we do like constants on the right side

I thought when I looked at this it was configurable to set left or right.